lib/jsdom/living/nodes/HTMLFormElement-impl.js

```diff
@@ -47,14 +47,53 @@ class HTMLFormElementImpl extends HTMLElementImpl {
     super._descendantRemoved.apply(this, arguments);
   }
 
+  _getElementNodes() {
+    if (!this.hasAttributeNS(null, "id")) {
+      return domSymbolTree.treeToArray(this, {
+        filter: node => this._filterElement(node)
+      });
+    }
+
+    const id = this.getAttributeNS(null, "id");
+    return domSymbolTree.treeToArray(this.ownerDocument, {
+      filter: node => {
+        if (!this._filterElement(node)) {
+          return false;
+        }
+
+        if (this.contains(node)) {
+          return true;
+        }
+
+        if (
+          node.hasAttributeNS(null, "form") &&
+          node.getAttributeNS(null, "form") === id
+        ) {
+          return true;
+        }
+
+        return false;
+      }
+    });
+  }
+
+  _filterElement(node) {
+    if (
```

Comment:
This function can be reduced to `return isListed(node) && (node._localName !== "input" || node.type !== "image");`

At that point it might be simpler to inline into into the above.

Note also that line lengths can be up to 120 characters so there's no need for this kind of breaking.